mmv1/third_party/terraform/website/docs/guides/provider_versions.html.markdown

```diff
@@ -39,7 +34,7 @@ features.
 -> In short, using `google-beta` over `google` is similar to using `gcloud beta`
 over `gcloud`. Features that are exclusively available in `google-beta` are GCP
 features that are not yet GA, and they will be made available in `google` after
-their GA launch.
+they are in General Availability.
```

Comment:
```suggestion
they go to GA.
```

nit: We've introduced GA earlier in the page and used it numerous times, it's more consistent to use here imo.

Reply:
Fixed